test/contracts/AuctionRegistrar.cpp

```diff
@@ -32,7 +32,7 @@
 #include <string>
 #include <optional>
 
-using namespace std;
+
```

Comment:
```suggestion
```

Please go over all of the files and remove newly introduced newlines such as this.